src/ML-POD/compute_podfit.cpp

```diff
@@ -0,0 +1,1564 @@
+/***************************************************************************
+               CESMIX-MIT Project
+
+ Contributing authors: Ngoc-Cuong Nguyen (cuongng@mit.edu, exapde@gmail.com)
+ ***************************************************************************/
+
+#ifndef __COMPUTEPODFIT
```

Comment:
what is this construct used for?

Reply:
I don't think it was used for anything.
We converted this compute to `fitpod_command.cpp` and removed the construct.